pages/contacts.html

```diff
@@ -0,0 +1,59 @@
+<div class="container-fluid">
+      <div class="row" id="contacts">
+          
+            <div class="col-xs-12 col-sm-12 col-md-12 text-center" style="margin-top: 5%">
+                <h1 class="head-one">CONTACTS</h1>                          
```

Comment:
Looks like all uppercase text in headers used here as a style. So you can use normal case in HTML: `<div>Contacts</div>` and apply CSS style `text-transform: uppercase;`